settings.py

```diff
@@ -0,0 +1,7 @@
+def init():
```

Comment:
do we need the init() method here?
All these path variables are constant so simple declaring them should suffice? similar to the globals.py